packages/mouse/dev/index.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  createMousePosition,
+  createMouseOnScreen,
+  createMouseToElement,
+  createMouseInElement
+} from "../src";
+import { Component, createSignal, Show } from "solid-js";
+import { render } from "solid-js/web";
+import createRaf from "@solid-primitives/raf";
```

Comment:
Not that this matters but should keep it as createRAF for consistency with the actual primitive. :-p